### DISCUSSION TITLE
Controller vs RestController

### DISCUSSION THREAD

**Discussion by giscus** [Cyma-s/Cyma-s.github.io#34] [General]
# Controller vs RestController

@Controller @Controller  는  @Component  클래스가 특화된 것으로, classpath scanning 을 통해 구현 클래스들을 자동으로 감지하도록 해준다. 보통  @Controller  와  @RequestMapping …

https://cyma-s.github.io/controller-vs-restcontroller/

**Comment by gitchannn** [comment]
좋은데! 그냥 컨트롤러를 전통적으로 사용할때 viewResolver 통해서 템플릿 찾는다는 내용도 추가해주면 더 좋을 거 같아

**Reply by Cyma-s** [reply]
오 굿굿!! 좋은 피드백 감사합니다~~~~😆
